### PR TITLE
Extract data from founders name plus simplified serialiser

### PR DESCRIPTION
**Pull request by IegorSoboliev**

### CODE REVIEW

File: business_register/converter/uo.py [hunk 1]
```diff
@@ -1,26 +1,20 @@
...
 from business_register.converter.business_converter import BusinessConverter
-from business_register.models.company_models import(
+from business_register.models.company_models import (
     Assignee, BancruptcyReadjustment, Bylaw, Company, CompanyDetail, CompanyToKved,
     CompanyToPredecessor, CompanyType, ExchangeDataCompany, FounderFull, Predecessor,
     Signer, TerminationStarted
 )
-from data_ocean.models import Status
-from django.conf import settings
+from data_converter import settings_local
```
Comment (roman-tiukh) [inline business_register/converter/uo.py:9]: bad settings import
```suggestion
from django.conf import settings
```

File: business_register/converter/uo.py [hunk 2]
```diff
@@ -1,26 +1,20 @@
...
+from data_ocean.converter import BulkCreateUpdateManager
+from data_ocean.utils import cut_first_word, format_date_to_yymmdd, get_first_word
 
 
 class Parser(BusinessConverter):
     LOCAL_FILE_NAME = settings_local.LOCAL_FILE_NAME_UO
     LOCAL_FOLDER = settings_local.LOCAL_FOLDER
```
Comment (roman-tiukh) [inline business_register/converter/uo.py:16]:
```suggestion
    LOCAL_FILE_NAME = settings.LOCAL_FILE_NAME_UO
    LOCAL_FOLDER = settings.LOCAL_FOLDER
```

File: business_register/serializers/company_serializers.py [hunk 3]
```diff
@@ -1,33 +1,17 @@
...
-
-class CompanyTypeSerializer(serializers.Serializer):
-    name = serializers.CharField(max_length=100)
-
-
-class AssigneeSerializer(serializers.Serializer):
-    name = serializers.CharField(max_length=100)
+from business_register.models.company_models import (
+    BancruptcyReadjustment, CompanyDetail,
+    ExchangeDataCompany, TerminationStarted, Company, FounderFull)
```
Comment (roman-tiukh) [inline business_register/serializers/company_serializers.py:5]:
```suggestion
    ExchangeDataCompany, TerminationStarted, Company, FounderFull
)
```

File: business_register/converter/uo.py (hunk 1 above)
Comment (IegorSoboliev) [reply business_register/converter/uo.py:9]: Done. That is not my code). Maybe, you could organize a tech-talk for all about using settings))

File: business_register/serializers/company_serializers.py (hunk 3 above)
Comment (IegorSoboliev) [reply business_register/serializers/company_serializers.py:5]: Done

**Pull request merged**